alpakaConfig.cmake

```diff
@@ -431,8 +431,10 @@ IF(ALPAKA_ACC_GPU_CUDA_ENABLE)
             SET(ALPAKA_CUDA_VERSION "${CUDA_VERSION}")
             IF(CUDA_VERSION VERSION_LESS 9.0)
                 SET(ALPAKA_CUDA_ARCH "20" CACHE STRING "GPU architecture")
-            ELSE()
+            ELSEIF(CUDA_VERSION VERSION_LESS 10.3)
                 SET(ALPAKA_CUDA_ARCH "30" CACHE STRING "GPU architecture")
+            ELSE()
+                SET(ALPAKA_CUDA_ARCH "60" CACHE STRING "GPU architecture")
```

Comment:
I am not sure if this is the most defensive thing we can do. The release notes only say that sm_30 will be removed. The others are only deprecated. We could use `sm_35` as new default.

Reply:
I agree with Benjamin, do not assume removal yet. Also, the deprecation period could go on for more than one release.

Reply:
```suggestion
                SET(ALPAKA_CUDA_ARCH "35" CACHE STRING "GPU architecture")
```

Reply:
Yes, also discussed offline with @psychocoderHPC that setting to `sm_35` is better.